test/unit/spec/Util_spec.js

```diff
@@ -39,6 +39,7 @@ define(['util/Util'], function (Util) {
         };
         Util.transformErrorXHR(xhr);
         expect(xhr.responseJSON.errorSummary).toEqual('errorSummary from errorCauses');
+        expect(xhr.responseJSON.errorCauses).toBeDefined();
```

Comment:
can we check that the object is the same as our ```errorCauses``` from line 31?

Second thing: can we add one more test with an invalid errorCode (like ```errorCode: "1212AB"```) instead of an undefined one? So we can test both scenarios

Reply:
yepp!